libs/blocks/preflight/panels/seo.js

```diff
@@ -110,8 +110,13 @@ async function checkDescription() {
 }
 
 async function checkBody() {
+  const nonContentEls = '#preflight, .picture-meta, aem-sidekick';
   const result = { ...bodyResult.value };
-  const { length } = document.documentElement.innerText;
+  let documentInnerText = document.documentElement.innerText;
```

Comment:
This seems to work well.

I personally nit would be that this type of manipulation feels safer/cleaner performed on a clone DOM. And instead of replacing nonContentEl's innerText, we can just remove these elements to avoid any unexpected innerText behavior.

`
const clone = document.body.cloneNode(true);
clone.querySelectorAll(nonContentEls).forEach(el => el.remove());
const { length } = clone.innerText.replace(/\n/g, '').trim();
`